resources/assetstore/templates/tests/rbac.yaml

```diff
@@ -27,7 +27,7 @@ rules:
     nonResourceURLs: []
   - apiGroups: ["assetstore.kyma-project.io"]
     resources: ["buckets", "clusterbuckets", "assets", "clusterassets"]
-    verbs: ["create", "get", "delete"]
+    verbs: ["create", "get", "delete", "watch", "list", deletecollection]
```

Comment:
![image](https://user-images.githubusercontent.com/17271979/67149489-0ee9a280-f2ac-11e9-87be-9292f5ed7f53.png)
looks like a typo

Reply:
it's one of the `API request verbs`:
https://kubernetes.io/docs/reference/access-authn-authz/authorization/#review-your-request-attributes;